oct/pages/models/add_address_page.py

```diff
@@ -0,0 +1,53 @@
+from pages.base.page import BasePage
+from pages.base.elements import RadioButton, Form, InputField, RadioButtonGroup, DropDown
+
+
+class DefaultAddress(RadioButtonGroup):
+    contains = {
+        "yes": {
+            "locator": ("XPATH", '//*[@id="content"]/form/fieldset/div[10]/div/label[1]/input'),
+            "class": RadioButton,
+        },
+        "no": {
+            "locator": ("XPATH", '//*[@id="content"]/form/fieldset/div[10]/div/label[2]/input'),
```

Comment:
Use a relative path, not an absolute one. This is almost the same as its parent element (https://github.com/ITA-Dnipro/dp-195-taqc-python/pull/13/files#diff-4c02beb94e979dbc036c36794765b46a6e6e120caf5ec651237ebbfd0d002e96R30)

Reply:
your truth, fixed